.travis/install-dependencies.sh

```diff
@@ -24,17 +24,19 @@ if [[ "$PYTHON" == "2.7" ]] && [[ "$platform" == "linux" ]]; then
   wget https://repo.continuum.io/miniconda/Miniconda2-4.5.4-Linux-x86_64.sh -O miniconda.sh -nv
   bash miniconda.sh -b -p $HOME/miniconda
   export PATH="$HOME/miniconda/bin:$PATH"
-  pip install -q cython==0.29.0 cmake tensorflow gym==0.10.11 opencv-python pyyaml pandas==0.23.4 requests \
+  pip install -q cmake gym==0.10.11 opencv-python-headless pyyaml pandas==0.23.4 requests \
```

Comment:
what is `opencv-python-headless`?

Reply:
On that note, we really ought to get rid of the opencv dependency for rllib. It's so heavyweight and we use it in so few places.

Reply:
It's the version of opencv that is not heavyweight (it doesn't link against libX and all this stuff). This just makes it much less frustrating (I always get this libSM error otherwise).

Yeah ideally we would get rid of it, I tried before and it came back somehow ;) But this headless version should be ok.